rosidl_generator_cpp/test/test_interfaces.cpp

```diff
@@ -540,7 +540,7 @@ TEST(Test_messages, primitives_default) {
 #pragma GCC diagnostic pop
 #endif
   TEST_PRIMITIVE_FIELD_ASSIGNMENT(message, byte_value, 50, 255);
-  TEST_PRIMITIVE_FIELD_ASSIGNMENT(message, char_value, 100, CHAR_MAX);
+  TEST_PRIMITIVE_FIELD_ASSIGNMENT(message, char_value, 100, UCHAR_MAX);
```

Comment:
@dirk-thomas hmm, `UCHAR_MAX != UINT8_MAX` here? `UCHAR_MAX == 127`?

Reply:
I would expect `UCHAR_MAX` to be `255`?

Reply:
Why not using `UINT8_MAX` then, like everywhere else?

Reply:
Sounds good to me: 6f9c6a6d0e8650e9c5c146920d943c202c536694